cmd/snap-seccomp/main.go

```diff
@@ -135,6 +136,11 @@ package main
 //		return htobe64(val);
 //}
 //
+//static int mygetgrnam_r(const char *name, struct group *grp,char *buf,
```

Comment:
Curious, why do we need this wrapper? Doesn't cgo find `getgrnam_r` without it? From a quick test (on amd64/zesty only) it seems there is no harm when I remove it.

Reply:
@mvo5 - again, I literally copied this from upstream's golang code. I did not analyze it extensively or try to improve it cause I felt it important to have it operate exactly like upstream when we switch over. If people want me to reimplement this, I'm not sure I see the benefit because we'll lose all that when we switch over. user.Lookup already exists, user.LookupGroup does not yet, but it will.

Reply:
Thanks for this explanation. Sorry, I was not aware that you answered this already :)

Reply:
@mvo5 - well, the 'again' wasn't so much for this portion, but for other parts-- but the rationale was the same in my head. Plus that happened in 3804 (which this is based off of), so easy to miss. :)